src/vs/editor/common/controller/cursorCollection.ts

```diff
@@ -182,24 +152,15 @@ export class CursorCollection {
 		if (this.lastAddedCursorIndex >= removeIndex + 1) {
 			this.lastAddedCursorIndex--;
 		}
-		this.secondaryCursors[removeIndex].dispose(this.context);
-		this.secondaryCursors.splice(removeIndex, 1);
-	}
-
-	private _getAll(): Cursor[] {
-		let result: Cursor[] = [];
-		result[0] = this.primaryCursor;
-		for (let i = 0, len = this.secondaryCursors.length; i < len; i++) {
-			result[i + 1] = this.secondaryCursors[i];
-		}
-		return result;
+		this.cursors[removeIndex + 1].dispose(this.context);
+		this.cursors.splice(removeIndex + 1, 1);
 	}
 
 	public normalize(): void {
-		if (this.secondaryCursors.length === 0) {
+		if (this.cursors.length === 1) {
 			return;
 		}
-		let cursors = this._getAll();
+		let cursors = [...this.cursors];
```

Comment:
stack overflow

Reply:
Sorry, this is muscle memory. I think I need a lint rule to remind me of that.